nameable.rb

```diff
@@ -0,0 +1,6 @@
+# Nameable class
+class Nameable
+  def correct_name
+    raise NotImplementedError
+  end
+end
```

Comment:
[OPTIONAL] Make sure you completely removed the Microverse [logo](https://github.com/Okechukwu-muokwugwo/OOP_school_library/blob/decorate_class/murple_logo.png) (img) from md file 👍🏼